model/models/nn.py

```diff
@@ -0,0 +1,25 @@
+"""
+Neural Network package (we needed a name for the file...)
+"""
+
+from tensorflow import keras
+
+
+def create_model(input_size, output_size):
+    """[summary]
+
+    Returns:
+        [type]: [description]
+    """
```

Comment:
Undocumented function. My mistake. This will be commented with the reviewers' correction commits.